### PR TITLE
Update text_classification.ipynb

### PR DESCRIPTION
**Pull request by ymodak**
Removed experimental api docs for TextVectorization layer

### CODE REVIEW

**Review by MarkDaoust — APPROVED** [review]

**Comment by MarkDaoust** [comment]
Thanks/

**Comment by lamberta** [comment]
Looks like the CI is failing.
Please format notebooks with [nbfmt](https://github.com/tensorflow/docs/blob/master/tools/tensorflow_docs/tools/nbfmt/__main__.py) and use [nblint](https://github.com/tensorflow/docs/blob/master/tools/tensorflow_docs/tools/nblint/__main__.py) to check style. Thank you

```
$ python3 -m pip install -U --user git+https://github.com/tensorflow/docs

$ python3 -m tensorflow_docs.tools.nbfmt notebook.ipynb
$ python3 -m tensorflow_docs.tools.nblint --arg=repo:tensorflow/docs notebook.ipynb
```

**Comment by 8bitmp3** [comment]
> Looks like the CI is failing.

@lamberta @ymodak I can assist remotely to save some ⌚ 👍 🕺 One mom

**Comment by 8bitmp3** [comment]
```diff
-         "from tensorflow.keras import preprocessing\n",
+         "from tensorflow.keras import preprocessing"
```

Since that line became the last line in the cell after the changes, the presence of `\n` made the notebook break unfortunately. I fixed it for you. Best to edit the notebook in Jupyter/Colab or similar @ymodak 👍

File: site/en/tutorials/keras/text_classification.ipynb [hunk 1]
```diff
@@ -121,7 +121,6 @@
         "from tensorflow.keras import layers\n",
         "from tensorflow.keras import losses\n",
         "from tensorflow.keras import preprocessing\n",
```
Comment (8bitmp3) [inline site/en/tutorials/keras/text_classification.ipynb:123]:
```suggestion
        "from tensorflow.keras import preprocessing"
```

**Comment by ymodak** [comment]
@8bitmp3 Thank you for making the changes and merging it. 👍🏼

**Pull request merged**